tests/lib.test.js

```diff
@@ -15,6 +15,15 @@ test('Gets datapackage.json (as extended)', async t => {
   t.is(dpjson.resources.length, 4)
 })
 
+test('Gets datapackage.json signedUrl', async t => {
+  let res = await api.getPackageFile(
+      'admin', 'private-package', 'datapackage.json', 'token')
+  let dpjson = await res.json()
+  t.is(dpjson.name, 'demo-package')
+  t.is(dpjson.datahub.findability, 'published')
```

Comment:
why this is "published"? Shouldn't it be "private"?

Reply:
ah, I guess you're reusing existing dp.json from fixtures.. Maybe it worth to create a new one - just duplicate this and have `findability='private'`